Formula/pastebinit.rb

```diff
@@ -25,6 +25,9 @@ def install
   end
 
   test do
-    system "date | pastebinit"
+    url = shell_output("echo \"Hello world at $(date)\" | pastebinit").chomp
```

Comment:
Should this use `pipe_output` with the optional `input` argument instead of an inline `|`?